Sources/UI/RequestsViewController.swift

```diff
@@ -133,41 +145,80 @@ class RequestsViewController: WHBaseViewController {
             self.show(requestDetailVC, sender: self)
         }
     }
-    
+
+    func modelIndex(fromPresentationIndex index: Int) -> Int {
+        return items.count - index - 1
+    }
+
+    func presentationIndex(fromModelIndex index: Int) -> Int {
+        return items.count - index - 1
+    }
```

Comment:
Would it be worth using UIKit's [UIDataSourceTranslating](https://developer.apple.com/documentation/uikit/uidatasourcetranslating) instead?
(not sure it's worth using it, but still worth a look in case you didn't know about it)

Reply:
I think that's for UITableView and UICollectionView to deal with drag & drop that requires adding a phantom item that doesn't exist in the data source.